docs/dev_guide/diagnostic_ref/dpct1043.rst

```diff
@@ -22,4 +22,54 @@ information and uses a different type than the original code.
 Suggestions to Fix
 ------------------
 
-Adjust the generated code.
\ No newline at end of file
+Adjust the generated code.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo() {
+      int version;
+      cudaDriverGetVersion(&version);
+      if (version >= 12000) {
+        ...
+      } else {
+        ...
+      }
+    }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo() {
+      int version;
+      /*
+      DPCT1043:0: The version-related API is different in SYCL. An initial code was
+      generated, but you need to adjust it.
+      */
+      version = dpct::get_current_device().get_major_version();
+      if (version >= 12000) {
+        ...
+      } else {
+        ...
+      }
+    }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo() {
+      int version;
+      version = dpct::get_current_device().get_major_version();
+      if (version >= 1) {

```

Comment:
//add comment to tell what version in L68,  and what does 1 mean in L69.

Reply:
done